tools/generator/CodeGenerator.cs

```diff
@@ -626,7 +626,7 @@ public string GetOutputName (string s)
 			if (s == "System.Void")
 				return "void";
 			if (s.StartsWith ("params "))
-				return "params " + GetOutputName (s.Substring (6));
+				return "params " + GetOutputName (s.Substring (7));
```

Comment:
Where is `7` coming from? Presumably it's `"params ".Length`.

Assuming that `7` *is* coming from a string length, could you lease replace `7` with e.g. `"params ".Length`, so that the source value is obvious?